airgun/views/host.py

```diff
@@ -240,6 +240,7 @@ class host(SatTab):
         inherit_deploy_option = ToggleButton(
             locator=".//div[label[@for='compute_resource_id']]//button")
         deploy = FilteredDropdown(id='host_compute_resource')
+        compute_profile = FilteredDropdown(id='s2id_host_compute_profile_id')
```

Comment:
I'm on snap 17 and I don't see "Compute Profile" between "Deploy On" and "Lifecycle Environment".
Is there anything specific that must be done for this one to appear?

Reply:
It only comes, when you select `Deploy` on a Compute Resource

Reply:
OK, figured that out. When you go to Compute Profiles page (there are some by default), you will have list of compute resources, and ability to change VM attributes of that compute profile in context of specific compute resource (at least for Azure these are VM attributes). 

Once you have compute profile with VM attribute for compute resource, then when you select this compute resource when creating host, new option will appear.

I'm not sure if we have a way in Airgun to specific complex visibility logic like that, so let's just add this here and hope for the best.